app/src/main/java/es/wolfi/passman/API/Core.java

```diff
@@ -48,7 +48,7 @@
     protected static String username;
     protected static String password;
     protected static String version_name;
-    protected static int version_number = 0;
+    protected static int versionNumber = 0;
```

Comment:
This naming scheme is inconsistent with the rest of the class properties, i believe snake_case is more appropiate to visually distinguish them from methods.

Reply:
changed it to snake case with 1a24073c528a77cb43e1833ee8f09d18d26c6406